README.md

```diff
@@ -49,10 +44,6 @@ Some secrets are required to deploy infrastructure into AWS and configure the in
 
  The AWS ACM ARN for the SSL certificate for the Cellenics domain name.
 
-- ACM_CERTIFICATE_ARN_STAGING  (optional, used only if the deployment is not on a production environment)
-
-  The AWS ACM ARN for the SSL certificate for the staging Cellenics domain name.
-
```

Comment:
Is there an alternative approach for production/staging being on different domains (as they are for HMS)?